cmd/mms/commands.go

```diff
@@ -51,7 +51,6 @@ func postEvent(hubs []mms.ProductionHub) func(*cli.Context) error {
 	return func(c *cli.Context) error {
 		productEvent := mms.ProductEvent{
 			Product:       c.String("product"),
-			ProductSlug:   c.String("product-slug"),
```

Comment:
Is this variable being removed? If so, it should be removed in the interface too.

Reply:
I think we should remove it, having a mandatory additional name (in addition to the file/object name) was mostly additional work for the maintainer. The "Product" should do. Perhaps we might introduce something like "Description" later.

Reply:
Where is it we need to remove it? I tried to grep for all ProductSlug occurences and remove those, maybe I have overseen some?